bcipy/helpers/stimuli.py

```diff
@@ -30,6 +30,8 @@ class InquirySchedule(NamedTuple):
     colors: List[List[str]]
 
 # TODO: since we have a querying this should replace the other generators
```

Comment:
I'm not sure I understand this TODO. Do you? Is this something we could move to a ticket?

Reply:
I don't understand it. I'll remove for now, and we can make a ticket if you want